apps/web/cypress/tests/notifications-editor.spec.ts

```diff
@@ -280,6 +280,32 @@ describe('Notifications Creator', function () {
     cy.getByTestId('emailSelector').click();
     cy.get('#codeEditor').contains('Hello world code {{name}} <div>Test</div>');
   });
+
+  it.only('should redirect to dev env for edit template', async function () {
```

Comment:
```suggestion
  it('should redirect to dev env for edit template', async function () {
```

So we can have our test running :)